From: Jo-Philipp Wich <jo@mein.io>
Date: Sun, 14 Aug 2016 22:18:53 +0200
Subject: tools: make_ext4fs: support creating empty filesystem images

Update make_ext4fs to latest git head in order to support creating empty
filesystem images by making the source directory argument optional.

Signed-off-by: Jo-Philipp Wich <jo@mein.io>

diff --git a/tools/make-ext4fs/Makefile b/tools/make-ext4fs/Makefile
index 553e6cd752f396377d3ce81738223c1d685fc747..a880dd4c57fd4bb34290c512565675d3dcd0da07 100644
--- a/tools/make-ext4fs/Makefile
+++ b/tools/make-ext4fs/Makefile
@@ -8,14 +8,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=make-ext4fs
-PKG_VERSION:=2016-05-05
+PKG_VERSION:=2016-08-14
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://git.openwrt.org/project/make_ext4fs.git
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_VERSION:=3af931b0b6545757aaeadb13f9bb9da0296fc172
+PKG_SOURCE_VERSION:=484903e4332be6c317f531b008cb2a841a18c506
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
-PKG_MIRROR_MD5SUM:=2c4161638d4ff3343e250321126d29e6
+PKG_MIRROR_MD5SUM:=b80d6ddcd6f6925e2f5b80775bb96f3e
 PKG_CAT:=zcat
 
 HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/$(PKG_NAME)-$(PKG_VERSION)
